ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -100,8 +104,12 @@ public struct Avatar: View, TokenizedControlView {
 
     public var body: some View {
         let style = state.style
+        let size = state.size
+        let activityStyle = state.activityStyle
         let presence = state.presence
-        let shouldDisplayPresence = presence != .none
+        let shouldDisplayActivity = activityStyle != .none
```

Comment:
Related to the comment @edjamesmsft made in the tokens, but we should be checking here if there's even anything to display.